packages/networks/bin/localhost.js

```diff
@@ -0,0 +1,83 @@
+import fs from 'fs-extra'
```

Comment:
Can you add a quick note in the README how the `localhost` network is different from he other ones?

Reply:
yep added